packages/shared/notionable-hooks/src/use-wallet.ts

```diff
@@ -43,6 +46,91 @@ export function useWalletBalanceInputCheck(
   };
 }
 
+function useApyValues(tradeType: string | undefined) {
+  // create a apyData object with a type of a Record with key of string and value of string
+  const apyData: Record<string, string> = {};
+  const {
+    yields: { fCashLend, variableLend, liquidity, fCashBorrow, variableBorrow, leveragedLiquidity },
+    getMax,
+    getMin,
+  } = useAllMarkets();
+
+  if(tradeType === 'LendFixed') {
```

Comment:
Can you format all the APYs with 3 decimal places?

Also sometimes you use `toPrecision` instead of `formatNumberAsPercent`, we should use the format method to stay consistent.

Also why is there no space after the if in your statements? You should have the `esbenp.prettier-vscode` extension installed. Otherwise I always get a bunch of linter changes when I update any of your files.

Reply:
Can we also wrap this in a `useMemo`? I feel like it will get called a lot since it is inside the `deposit-input` component